tests/cnf/core/network/metallb/internal/frr/frr.go

```diff
@@ -313,6 +314,34 @@ func FetchBGPConnectTimeValue(frrk8sPods []*pod.Builder, bgpPeerIP string) (int,
 	return 0, fmt.Errorf("no BGP neighbor data found for peer %s", bgpPeerIP)
 }
 
+// FetchBGPRemoteAS fetches and returns the remoteAS value for the specified BGP peer.
+func FetchBGPRemoteAS(frrk8sPods []*pod.Builder, bgpPeerIP string) (int, error) {
```

Comment:
iiuc this currently does not go over all of the pods, but returns the first remoteAS it finds, should we have this accept a single frrk8spod, and then iterate over all of them outside?

Reply:
changed to check  each frr node for the correct AS number